src/manifests/build_manifest.py

```diff
@@ -22,6 +22,8 @@
   version: string
   platform: linux, darwin or windows
   architecture: x64 or arm64
+  distribution: tar, zip, and more
```

Comment:
Be specific here instead of more. `tar, zip and rpm`

Reply:
I will update.